builder/vmware/common/host_ip_ifconfig.go

```diff
@@ -57,7 +57,7 @@ func ipaddress(device string) (string, error) {
 	re := regexp.MustCompile(`inet[^\d]+([\d\.]+)/`)
 	matches := re.FindStringSubmatch(stdout.String())
 	if matches == nil {
-		return "", errors.New("IP not found in ip a output...")
+		return "", errors.New("ip address not found in ip output")
```

Comment:
Capitalisation issue, and I would suggest fencing `ip a` since it's the command that was invoked.

```suggestion
		return "", errors.New("could not find a usable IP address in `ip address show dev %s` output", device)
```

Reply:
Resolved in 67054cd.